src/components/common/FooterBar.js

```diff
@@ -8,8 +8,8 @@ function FooterBar() {
   ];
 
   return (
-    <footer className="bg-blue-1000 ">
-      <span className="text-blue-500 uppercase text-s mt-6">Logo</span>
+    <footer className="bg-primary-dark flex justify-between ">
+      <span className="text-gradient-dark-blue uppercase text-s mt-6">Logo</span>
```

Comment:
@Domicaren1 I do not think this is the correct shade of blue. Please check the tailwind.config file for a more accurate shade. Don't worry about the links - the team decided not to have those pages, so those links will probably be deleted.

Reply:
@msarit It's corrected already ma